index.html

```diff
@@ -1,10 +1,136 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Rekupe, the digital agency</title>
 </head>
 <body>
 
 </body>
+<header>
+    <nav>
+        <ul>
+            <li><a href="#case_studies">Case studies</a></li>
+            <li><a href="#services">Services</a></li>
+            <li><a href="#contact_us">Contact us</a></li>
+        </ul>
+    </nav>
+    <div class="intro">
+        <h1>Your website is an investment in the success of your business.</h1>
+        <h2>We are here to help you succeed.</h2>
+        <button>Let's work together</button>
+        <div class="squigle_wrapper">
+            <img src="assets/squiggle_line_stripe.svg" />
+        </div>
+    </div>
+
+
+</header>
+<main>
+    <section class="introduction">
+        <div class="introduction_inner">
+            <p>Rekupe is a digital agency in Los Angeles.</p>
+            <p>We use strategic UX, SEO optimization and beautiful visual designs to help your business grow.</p>
+            <p>We see every project as a fresh opportunity, pushing ourselves to create new digital experiences.</p>
+            <p id="contact_shortcut">Have a question? <a href="#contact_us">Click here</a> to contact us!</p>
+        </div>
+    </section>
+    <section class="case_studies" id="case_studies">
+        <div class="case_studies_inner">
+            <article>
+                <div class="article_text">
+                    <h2>Rolling Stone</h2>
+                    <p>You know Rolling Stone. It's the definitive voice in music, politics and culture. It's a global icon.</p>
+                    <p><a href="#">View project</a></p>
+                </div>
+                <span class="image_wrapper"><img src="assets/work-image-01.jpeg" alt="Laptop and Rolling Stone magazine"/> </span>
+            </article>
+            <article class="reverse">
+                <div class="article_text">
+                    <h2>Variety</h2>
+                    <p>Variety is an industry leader for breaking entertainment news, reviews and box office results with archive of content dating back to 1905.</p>
+                    <p><a href="#">View project</a></p>
+                </div>
+                <span class="image_wrapper"><img src="assets/work-image-02.jpeg" alt="Laptop and Variety magazine"/> </span>
+            </article>
+            <article>
+            <div class="article_text">
+                <h2>Worth</h2>
+                <p>A global media platform connecting a community that embraces worth beyond wealth.</p>
+                <p><a href="#">View project</a></p>
+            </div>
+            <span class="image_wrapper"><img src="assets/work-image-03.jpeg" alt="Tablet showing magazines"/> </span>
+            </article>
+        </div>
+    </section>
+    <section class="tiles" id="services">
+        <div class="tiles_inner">
+            <article class="tile">
```

Comment:
Als ik het venster smaller maak vind ik dat de kaartjes wel wat breder mogen zijn. Ze zouden in principe de beschikbare ruimte in mogen nemen

![Scherm­afbeelding 2023-01-21 om 12 08 10](https://user-images.githubusercontent.com/23332341/213864207-b4761963-4308-4150-88a1-c6681980aa03.jpg)